### ISSUE TITLE
[BUG] Incorrect minimum Y level height

### ISSUE THREAD

**Issue by ginsm** [ginsm/forgotten-graves#24]
**Describe the bug**
L63-65 of `Graves.java` has the following code:

```
if (pos.getY() < 0) {
  pos = new BlockPos(pos.getX(), 10, pos.getZ());
}
```

This is an issue because the world has been moved down to y-64.

**Forgotten Graves Version**
v2.0.0

**Expected behavior**
It should generate the grave at `y-60` as opposed to `y10` and have a minimum Y level of `y-64` rather than `y0`.

**Comment by ginsm** [comment]
This will be fixed in the next update (which should release later today or tomorrow).

**Comment by ginsm** [comment]
This has been fixed in 06720b6 and 75e0bce. 

Closing.
